packages/launch/src/focused-launch/summary/index.tsx

```diff
@@ -32,13 +32,12 @@ import FocusedLaunchSummaryItem, {
 	LeadingContentSide,
 	TrailingContentSide,
 } from './focused-launch-summary-item';
-import { LAUNCH_STORE, SITE_STORE, Plan } from '../../stores';
+import { LAUNCH_STORE, SITE_STORE, Plan, PlanProduct, PLANS_STORE } from '../../stores';
```

Comment:
I think it's also fine to add on another line: `import type { Plan, PlanProduct } from '../../stores';`